src/Data/Fin/Properties.agda

```diff
@@ -491,7 +492,11 @@ splitAt-raise : ∀ m n i → splitAt m (raise {n} m i) ≡ inj₂ i
 splitAt-raise zero    n i = refl
 splitAt-raise (suc m) n i rewrite splitAt-raise m n i = refl
 
-inject+-raise-splitAt : ∀ m n i → [ inject+ n , raise {n} m ] (splitAt m i) ≡ i
+splitAt-join-⊎ : ∀ m n i → splitAt m (join-⊎ m n i) ≡ i
+splitAt-join-⊎ m n (inj₁ x) = splitAt-inject+ m n x
+splitAt-join-⊎ m n (inj₂ y) = splitAt-raise m n y
+
+inject+-raise-splitAt : ∀ m n i → join-⊎ m n (splitAt m i) ≡ i
```

Comment:
We should deprecate this name and call it `join-⊎-splitAt`/`join-⊎-splitAt`

Reply:
Agreed, will do.